sqlalchemy_ibmi/base.py

```diff
@@ -686,8 +688,8 @@ def __init__(self, **kw):
     # object which selects between Db2 and AS/400 schemas
     def initialize(self, connection):
         super().initialize(connection)
-        self.dbms_ver = getattr(connection.connection, 'dbms_ver', None)
-        self.dbms_name = getattr(connection.connection, 'dbms_name', None)
+        self.dbms_ver = connection.connection.getinfo(pyodbc.SQL_DBMS_VER)
+        self.dbms_name = connection.connection.getinfo(pyodbc.SQL_DBMS_NAME)
```

Comment:
We don't need DBMS_NAME, since we will only ever talk to one database (Db2 for i).